gen/definitions/classes.yaml

```diff
@@ -172,6 +172,111 @@ fvRsApMonPol:
 
 fvCtx:
   resource_name: "vrf"
+  ui_locations:
+    - "Tenants -> Networking -> VRFs"
+  sub_category: "Networking"
+  migration_version: 1
+  migration_blocks:
+    fvCtx:
+        tenant_dn: parent_dn
+        bd_enforced_enable : bd_enforcement
+        pc_enf_dir: policy_control_enforcement_direction
+        pc_enf_pref: policy_control_enforcement_preference
+    fvRsBgpCtxPol:
+        relation_fv_rs_bgp_ctx_pol: relation_to_bgp_timers.bgp_timers_name
+    fvRsCtxMonPol:
+        relation_fv_rs_ctx_mon_pol: relation_from_vrf_to_monitoring_policy.monitoring_policy_name
+    fvRsCtxToBgpCtxAfPol:
+        relation_fv_rs_ctx_to_bgp_ctx_af_pol: relation_from_vrf_to_bgp_address_family_context
+        tn_bgp_ctx_af_pol_name: relation_from_vrf_to_bgp_address_family_context.bgp_address_family_context_name
+        af: relation_from_vrf_to_bgp_address_family_context.address_family
+    fvRsCtxToEigrpCtxAfPol:
+        relation_fv_rs_ctx_to_eigrp_ctx_af_pol: relation_from_vrf_to_eigrp_address_family_context
+        tn_eigrp_ctx_af_pol_name: relation_from_vrf_to_eigrp_address_family_context.eigrp_address_family_context_name
+        af: relation_from_vrf_to_eigrp_address_family_context.address_family
+    fvRsCtxToEpRet:
+        relation_fv_rs_ctx_to_ep_ret: relation_from_vrf_to_end_point_retention_policy.end_point_retention_policy_name
+    fvRsCtxToExtRouteTagPol:
+        relation_fv_rs_ctx_to_ext_route_tag_pol: relation_from_vrf_to_l3out_route_tag_policy.l3out_route_tag_policy_name
+    fvRsCtxToOspfCtxPol:
+        relation_fv_rs_ctx_to_ospf_ctx_pol: relation_from_vrf_to_address_family_ospf_timers
+        tn_ospf_ctx_pol_name: relation_from_vrf_to_address_family_ospf_timers.ospf_timers_name
+        af: relation_from_vrf_to_address_family_ospf_timers.address_family
+    fvRsOspfCtxPol:
+        relation_fv_rs_ospf_ctx_pol: relation_to_ospf_timers.ospf_timers_name
+
+fvRsBgpCtxPol:
+  resource_name: "relation_to_bgp_timers"
+  ui_locations:
+    - "Tenants -> Networking -> VRFs -> Policy"
```

Comment:
Can we use a more specific UI location instead of pointing to the generic UI location? The same comment for the other resources which is using "Tenants -> Networking -> VRFs -> Policy".